src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -114,7 +114,7 @@ class Docker implements Serializable {
         
         public <V> V inside(String args = '', Closure<V> body) {
             docker.node {
-                if (docker.script.sh(script: "docker inspect -f . ${id}", returnStatus: true) != 0) {
+                if (docker.script.sh(script: "docker inspect --type image ${id}", returnStatus: true) != 0) {
```

Comment:
`-f .` is important to keep the output in the build log to a minimum.

Was it really the format argument which did not work in UCP, or just the lack of a type? If the latter, then you would want

    docker inspect -f . --type image ${id}